CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ All notable changes to this project will be documented in this file.  The format
 
 ## [Unreleased]
 
+### Changed
+* Update pinned version of Rust toolchain and dependencies.
```

Comment:
I think it makes sense to mention the version explicitly if we're adding a changelog entry, otherwise readers would have to search this commit through git history to find it (assuming it will change again in the future).

Reply:
Good point.  Done in [99176a4](https://github.com/casper-ecosystem/cargo-casper/pull/35/commits/99176a433a5de0ff811a3cac78aed03c327de7d0).